tools/install_prereqs.sh

```diff
@@ -56,8 +56,15 @@ echo "Dependencies are installed at $PREFIX"
 # Install deps with rbuild
 rbuild prepare -d $PREFIX -s develop
 
-# install onnx package for unit tests
-pip3 install onnx==1.10.2 numpy==1.21.6 typing==3.7.4 pytest==6.0.1 packaging==23.0
+if ! python3 -c 'import sys; assert sys.version_info >= (3, 10)' > /dev/null; then
+  python3.8 -m pip install protobuf==3.20.3 onnx==1.10.2 numpy==1.21.6 typing==3.7.4 pytest==6.0.1 packaging==23.0
+  export CMAKE_ARGS="-DONNX_USE_PROTOBUF_SHARED_LIBS=ON"
+  python3.10 -m pip install onnx==1.10.2
+  python3.10 -m pip install protobuf==3.20.3 numpy==1.21.6 typing==3.7.4 pytest==6.0.1 packaging==23.0
+else
+  # install onnx package for unit tests
+  pip3 install onnx==1.10.2 numpy==1.21.6 typing==3.7.4 pytest==6.0.1 packaging==23.0
 
-# pin version of protobuf in Python for onnx runtime unit tests
-pip3 install protobuf==3.20.0
+  # pin version of protobuf in Python for onnx runtime unit tests
+  pip3 install protobuf==3.20.0
+fi
```

Comment:
If you need to install different protobuf versions, you should do:

```
pip3 install onnx==1.10.2 numpy==1.21.6 typing==3.7.4 pytest==6.0.1 packaging==23.0
if ! python3 -c 'import sys; assert sys.version_info >= (3, 10)' > /dev/null; then
    CMAKE_ARGS="-DONNX_USE_PROTOBUF_SHARED_LIBS=ON" pip3 install protobuf==3.20.3
else
    pip3 install protobuf==3.20.0
fi
```

But I wonder if we can just use the same version of protobuf.

Reply:
Probably can its just a small version diff 3.20.* , i'll retry to default it all to 3.20.0